playground/docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '2.1'
+
+networks:
+  monitor-net:
+    driver: bridge
+
+services:
+  prometheus:
+    image: prom/prometheus:v2.44.0
+    container_name: prometheus
+    volumes:
+      - ./configs/prometheus:/etc/prometheus
+      - ./data/prometheus:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--storage.tsdb.retention.time=200h'
+      - '--web.enable-lifecycle'
+    restart: unless-stopped
+    expose:
+      - 9090
+    networks:
+      - monitor-net
+
+  prompush:
```

Comment:
rename to `prometheus-pushgateway` for consistency with https://github.com/ydb-platform/slo-tests/blob/main/.github/workflows/run-slo.yml#LL74C31-L74C53